SAMPLES.md

```diff
@@ -55,7 +55,7 @@ import SendbirdApp from '@sendbird/uikit-react/App';
 
 Here is a list of the essential components that you need to import before you start customizing chat. Note that the names of the components are changed as shown in the code below.
 
-Try [importing components on CodeSandbox](https://codesandbox.io/s/1-2-customization-basic-format-9skqq3).
+Try [importing components on StackBlitz](https://stackblitz.com/edit/sendbird-react-uikit-customized-app).
```

Comment:
nit:
1.
I want to suggest using this link instead
https://stackblitz.com/edit/sendbird-react-uikit-customized-app?file=src%2FCustomizedAppHOC.jsx

2.
It seems we can remove the `CustomizedAppHook.jsx` file
<img width="338" alt="image" src="https://github.com/sendbird/sendbird-uikit-react/assets/46333979/e2f32231-754d-4e5c-9622-0ebcd81477b7">

Reply:
I added HOC and HOOK options because I think they both have their uses cases. The idea to was to show you how you could use one or the other. We can stick with just HOC though if you'd like.

Reply:
ahah, I couldn't the difference, it looks fine to me. Thanks a ton!